com.unity.netcode.gameobjects/Tests/Runtime/Metrics/OwnershipChangeMetricsTests.cs

```diff
@@ -61,6 +61,10 @@ public IEnumerator TrackOwnershipChangeSentMetric()
             var ownershipChangeSent = metricValues.First();
             Assert.AreEqual(networkObject.NetworkObjectId, ownershipChangeSent.NetworkId.NetworkId);
             Assert.AreEqual(Server.LocalClientId, ownershipChangeSent.Connection.Id);
+
+            // The first metric is to the server(self), so its size is now correctly reported as 0.
```

Comment:
I'd either assert everything possible on both metric values, or, if only the last value is relevant, ignore the first one and do all the asserts on that last value.